databricks/koalas/plot.py

```diff
@@ -1118,6 +1118,77 @@ def _plot(data, x=None, y=None, subplots=False, ax=None, kind="line", **kwds):
     return plot_obj.result
 
 
+_backends = {}
+import importlib
+
+
+def _find_backend(backend: str):
+    """
+    Find a pandas plotting backend
+    """
+    # function copied from pandas.plotting._core
+
+    import pkg_resources  # Delay import for performance.
```

Comment:
Could we move this comment to the above of `import pkg_resources` to keep the consistency with other comments if there is no special reason ?